pkg/config/evaluation.go

```diff
@@ -189,6 +195,13 @@ func (s *StepCondition) evaluateV1(config StepConfig, utils piperutils.FileUtils
 		return false, nil
 	}
 
+	if s.OnlyActiveStepInStage {
+		// Used only in NotActiveConditions.
+		// Returns true if all previous steps are inactive, so step will be deactivated
+		// if it's the only active step in stage.
```

Comment:
maybe add one more line of comment 
for eg., sapCumulusUpload need not be active in a stage where other steps are not active

Reply:
done